.github/workflows/test.yml

```diff
@@ -52,11 +52,29 @@ jobs:
       - name: run checks & tests
         run: RUST_MATRIX=${{ matrix.rust }} cargo xtask run-checks ${{ matrix.test }}
 
+      - name: Install Grcov
+        run: cargo install grcov
```

Comment:
To speed up CI and save `grcov` binary in cache we can use the provided binary 

```yaml
    - name: Install grcov
      env:
        GRCOV_LINK: https://github.com/mozilla/grcov/releases/download
        GRCOV_VERSION: v0.8.13
      run: |
        curl -L "$GRCOV_LINK/$GRCOV_VERSION/grcov-x86_64-unknown-linux-musl.tar.bz2" |
        tar xj -C $HOME/.cargo/bin
```

Reply:
:+1: